packages/dds/matrix/src/matrix.ts

```diff
@@ -634,20 +632,14 @@ export class SharedMatrix<T = any>
 	}
 
 	private rebasePosition(
-		// eslint-disable-next-line import/no-deprecated
-		client: Client,
-		pos: number,
-		referenceSequenceNumber: number,
+		client: PermutationVector,
 		localSeq: number,
+		handle: number,
 	): number | undefined {
-		const { clientId } = client.getCollabWindow();
-		const { segment, offset } = client.getContainingSegment(
-			pos,
-			{ referenceSequenceNumber, clientId: client.getLongClientId(clientId) },
-			localSeq,
-		);
-		if (segment === undefined || offset === undefined) {
-			return;
+		const { segment, offset } = client.handleToSegment(handle);
+
+		if (segment.removedSeq !== undefined) {
+			return undefined;
 		}
```

Comment:
If we're rebasing for reconnect, and the current known status of the row/col is removed, I think we always drop the op?